app/steps/ui/deceased/otherNames/index.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const ValidationStep = require('app/core/steps/ValidationStep');
+const FieldError = require('app/components/error');
+const {get, set, isEmpty} = require('lodash');
+const FormatName = require('app/utils/FormatName');
+
+class DeceasedOtherNames extends ValidationStep {
+
+    static getUrl() {
+        return '/other-names';
+    }
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [{
+                key: 'deceasedMarriedAfterDateOnCodicilOrWill',
+                value: true,
+                choice: 'deceasedMarriedAfterDateOnCodicilOrWill'
+            }]
+        };
+        return nextStepOptions;
+    }
+
+    getContextData(req) {
+        const ctx = super.getContextData(req);
+        if (!ctx.otherNames) {
+            set(ctx, 'otherNames.name_0.firstName', '');
+            set(ctx, 'otherNames.name_0.lastName', '');
+        }
+        const formdata = req.session.form;
+        ctx.deceasedFullName = FormatName.format(formdata.deceased);
+        return ctx;
+    }
```

Comment:
Same thing I guess?

Reply:
agreed with David to leave template substitution as PA journey